### ISSUE TITLE
Feature request: Un-align declarations

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#35]
The Clean ABAP style guide suggests to not use chaining and not to align type declarations.

ABAP Cleaner provides the rule "Unchain into multiple statements", but any previous alignment is not removed. Could this be an option or another rule?

Example:

```
PARAMETERS:
  p_short  TYPE i,
  p_longer TYPE char20.
```

Result:

```
PARAMETERS p_short  TYPE i.
PARAMETERS p_longer TYPE char20.
```

Expected:

```
PARAMETERS p_short TYPE i.
PARAMETERS p_longer TYPE char20.
```

**Comment by fabianlupa** [comment]
Can you check if it's not automatically aligned after unchaining by the align declarations rule? You can see which rules applied in the preview window. I was also a bit surprised alignment across multiple statements is a feature here (https://github.com/SAP/abap-cleaner/issues/14#issuecomment-1554209714).

**Comment by ConjuringCoffee** [comment]
I should have mentioned this: The rule "Align declarations" is inactive for these examples.

**Comment by ConjuringCoffee** [comment]
You got it right, Jörg-Michael. 👍

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

in the next release, the "Align declarations" rule will allow you to specify how much alignment you want:

![image](https://github.com/SAP/abap-cleaner/assets/79639298/e65541a0-415c-4278-9652-8b551966cb8d)

You can select this independently for chains, non-chains, and structures (because Pretty Printer behaves differently for those, too). Everything that is not aligned can be condensed with the last option, thus removing spaces from any previous alignment. 

So, with "**align name, TYPE, LENGTH, VALUE etc.**" and activated "Condense …", you get:

```ABAP
    CONSTANTS: lc_any_price           TYPE ty_price           VALUE 1200,
               lc_other_price         TYPE ty_price           VALUE 1000,
               lc_num_contract_change TYPE ty_sequence_number VALUE 1,
               lc_start_date          TYPE ty_start_date      VALUE '20220312'.

    CONSTANTS lc_pi      TYPE p LENGTH 10 DECIMALS 10 VALUE '3.1415926536'.
    CONSTANTS lc_e       TYPE p LENGTH 8  DECIMALS 10 VALUE '2.718281828'.
    CONSTANTS lc_sqrt_2  TYPE p LENGTH 8  DECIMALS 4  VALUE '1.4142'.
    CONSTANTS lc_sqrt_32 TYPE p LENGTH 8  DECIMALS 10 VALUE '5.6568542495'.
```

With "**align name and TYPE**" and activated "Condense …":

```ABAP
    CONSTANTS: lc_any_price           TYPE ty_price VALUE 1200,
               lc_other_price         TYPE ty_price VALUE 1000,
               lc_num_contract_change TYPE ty_sequence_number VALUE 1,
               lc_start_date          TYPE ty_start_date VALUE '20220312'.

    CONSTANTS lc_pi      TYPE p LENGTH 10 DECIMALS 10 VALUE '3.1415926536'.
    CONSTANTS lc_e       TYPE p LENGTH 8 DECIMALS 10 VALUE '2.718281828'.
    CONSTANTS lc_sqrt_2  TYPE p LENGTH 8 DECIMALS 4 VALUE '1.4142'.
    CONSTANTS lc_sqrt_32 TYPE p LENGTH 8 DECIMALS 10 VALUE '5.6568542495'.
```

With "**align name only**" and activated "Condense …":

```ABAP
    CONSTANTS: lc_any_price TYPE ty_price VALUE 1200,
               lc_other_price TYPE ty_price VALUE 1000,
               lc_num_contract_change TYPE ty_sequence_number VALUE 1,
               lc_start_date TYPE ty_start_date VALUE '20220312'.

    CONSTANTS lc_pi TYPE p LENGTH 10 DECIMALS 10 VALUE '3.1415926536'.
    CONSTANTS lc_e TYPE p LENGTH 8 DECIMALS 10 VALUE '2.718281828'.
    CONSTANTS lc_sqrt_2 TYPE p LENGTH 8 DECIMALS 4 VALUE '1.4142'.
    CONSTANTS lc_sqrt_32 TYPE p LENGTH 8 DECIMALS 10 VALUE '5.6568542495'.
```

So the choice is now yours ;-)

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee and Fabian, 

this is now available in release 1.4.0! 

Kind regards,
Jörg-Michael
